test/unit/fact_value_test.rb

```diff
@@ -69,6 +69,22 @@ def setup
     assert_empty results
   end
 
+  test 'numeric searches should use numeric comparsion' do
+    host = FactoryGirl.create(:host)
+    FactoryGirl.create(:fact_value, :value => '64498',:host => host,
+                       :fact_name => FactoryGirl.create(:fact_name, :name => 'memory_mb'))
```

Comment:
I was under impression that our facter importer code does convert everything to strings. How can I achieve an integer in facts in runtime?